src/lib/y2partitioner/cli.rb

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2014 SUSE LLC.
+#  All Rights Reserved.
+
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 2 or 3 of the GNU General
+#  Public License as published by the Free Software Foundation.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+
+#  To contact Novell about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+Yast.import "CommandLine"
+
+module Y2Partitioner
+  # CLI support for storage-ng. It just say that there is no one.
+  class CLI
+    extend Yast::I18n
+
+    def self.run
+      cmdline_description = {
+        "id"   => "partitioner"
+      }
```

Comment:
Does this actually print something?

Reply:
yes, it prints all that is needed:
```
localhost:/home/jreidinger # yast2 partitioner help
YaST2 partitioner
-----------------

This YaST2 module does not support the command line interface.
```

for others I use help to suggest CLI tool, but for all partitioner functionality it is hard.